lib/asyncbox.js

```diff
@@ -230,7 +230,7 @@ async function waitForCondition (condFn, options = {}) {
     }
     // if there is an error option, it is either a string message or an error itself
     throw error
-      ? (_.isString(error) ? new Error(error) : error)
+      ? (typeof error === 'string' ? new Error(error) : error)
```

Comment:
just checking typeof is not enough to verify a variable is a string.

Reply:
how so? string is primitive type. template strings are still strings too

Reply:
if you need `Object#toString` support, I can add it, but doesn't seem like it accepts objects anyway. It's an either an `Error` object or a `string` in types:

https://github.com/jlipps/asyncbox/blob/ac95eb0d2722e2d6816980209f2586c3a94985ba/lib/asyncbox.js#L244-L251